### ISSUE TITLE
Adjust formatting for subheading in challenge

### ISSUE THREAD

**Issue by ndporter** [carpentries/instructor-training#1611]
### What is the problem?

The subheading "accessibility adaptation" in the [Test Your Working Memory Analog Version challenge box](https://carpentries.github.io/instructor-training/instructor/05-memory.html#test-your-working-memory) should be formatted to be comparable in size to the main challenge heading (not larger). I tried just changing it from an H2 to H3 but that didn't seem to change it. I'm not sure if there's a native solution using `sandpaper` or if this is a situation where we should hard-code visual formats (so I didn't make a PR) but it's jarring to run across.

### Location of problem (optional)

https://github.com/carpentries/instructor-training/blob/main/episodes/05-memory.md
